girder/events.py

```diff
@@ -44,10 +44,26 @@
 import contextlib
 import girder
 import threading
+from functools import wraps
+import warnings
 
 from girder.utility import config
 from six.moves import queue
 
+def deprecated_async(func):
+    """A decorator, that let's us keep our old API, but deprecate it"""
+    @wraps(func)
+    def inner(*args, **kwargs):
+        if 'async' in kwargs:
+            if 'asynq' in kwargs:
+                raise ValueError('cannot use both async and asynq '
+                                 'keyword arguments! the latter obsoletes the first.')
+            warnings.warn('async keyword argumnt is deprecated, '
+                          'use asynq instead', DeprecationWarning)
+            kwargs['asynchronous'] = kwargs.pop('async')
```

Comment:
Is this supposed to be "asynchronous"?

Reply:
No my bad, updating now, still trying to get my dev environment up on Windows.
Also I decided on asynq since that what I saw in a few other fixes. Though official convention is async_
We can change it to whatever we think might work.